multiqc/modules/skewer/skewer.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse logs from Skewer """
+
+from __future__ import print_function
+
+import os
+from collections import OrderedDict
+import logging
+import re
+from multiqc import config, BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Skewer """
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Skewer', anchor='skewer',
+                                            href="https://github.com/relipmoc/skewer",
+                                            info="is an adapter trimming tool specially designed for processing next-generation sequencing (NGS) paired-end sequences.")
+
+        self.sections = list()
+        self.skewer_data = dict()
+        self.skewer_readlen_dist = dict()
+
+        for f in self.find_log_files(config.sp['skewer'], filehandles=True):
+            self.parse_skewer_log(f)
+
+        headers = OrderedDict()
+        headers['pct_trimmed'] = {
+            'title': '% Trimmed',
+            'description': '% of reads trimmed',
+            'scale': 'RdYlGn-rev',
+            'max': 100,
+            'min': 0
+        }
+
+        self.general_stats_addcols(self.skewer_data, headers)
+
+        # Write parsed report data to a file
+        self.write_data_file(self.skewer_data, 'multiqc_skewer')
+        self.write_data_file(self.skewer_readlen_dist, 'multiqc_skewer_readlen_dist')
```

Comment:
So, interesting discussion point. In the cutadapt module (and others), I _don't_ export the continuous data that is parsed for the plots. This is vaguely because I was worried about it making very large files if 1000s of samples are processed. But if I'm honest I haven't thought about it a great deal.

On the one hand it's good to make the parsed data as accessible as possible for downstream applications, but on the other we don't want to bloat the reports too much for the 99% of people who aren't interested in this data.

Any thoughts?